sys/include/net/gnrc/sixlowpan/frag/stats.h

```diff
@@ -35,6 +35,8 @@ typedef struct {
                              *   reassembly buffer is full */
     unsigned frag_full;     /**< counts the number of events that there where
                              *   no @ref gnrc_sixlowpan_frag_fb_t available */
+    unsigned datagrams;     /**< reassembled datagrams */
+    unsigned fragments;     /**< total fragments of reassembled fragments */
 #if defined(MODULE_GNRC_SIXLOWPAN_FRAG_VRB) || DOXYGEN
```

Comment:
```suggestion
#endif /* MODULE_GNRC_SIXLOWPAN_FRAG_STATS */
#if defined(MODULE_GNRC_SIXLOWPAN_FRAG_VRB) || DOXYGEN
```

Reply:
I don't understand why you want to introduce unopened `#endif`s ...